.github/workflows/release.yml

```diff
@@ -294,7 +341,11 @@ jobs:
   publish-runtimes:
     runs-on: ubuntu-latest
     needs: ["publish-draft-release"]
-    if: needs.get-tag-version.outputs.tag-exists == 1
+    # We want to store the binaries also when it is not a version release. This is used
+    # in case such as providing binaries when creating a new tutorial version.
```

Comment:
I agree with "We want to store the binaries also when it is not a version release" and thus I agree with this code.

But I think we should start striving toward having the tutorials target a released version.

Reply:
Yes, you are right. We will remove this part once this is done